WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -119,6 +127,8 @@ public class BlockEditorScreen: ScreenObject {
     var switchToHTMLModeButton: XCUIElement { switchToHTMLModeButtonGetter(app) }
     var undoButton: XCUIElement { undoButtonGetter(app) }
     var unsavedChangesLabel: XCUIElement { unsavedChangesLabelGetter(app) }
+    var turnOnImageOptimization: XCUIElement { turnOnImageOptimizationGetter(app) }
+    var turnOffImageOptimization: XCUIElement { turnOffImageOptimizationGetter(app) }
```

Comment:
Same sentiment here:

```suggestion
    var turnOnImageOptimizationButton: XCUIElement { turnOnImageOptimizationButtonGetter(app) }
    var turnOffImageOptimizationButton: XCUIElement { turnOffImageOptimizationButtonGetter(app) }
```

Reply:
Suggestion applied in https://github.com/wordpress-mobile/WordPress-iOS/pull/22095/commits/88e54245ae1d1eafc80e92ce4ec51a4ae2820140.